.github/workflows/build.yml

```diff
@@ -99,7 +99,7 @@ jobs:
           submodules: 'true'
       - uses: ./.github/actions/install_toolchain
       - name: Run cargo tests
-        run: cargo test -vv --workspace --exclude node-executor --exclude phala-node
+        run: cargo test --tests -vv --workspace --exclude node-executor --exclude phala-node
```

Comment:
`--tests` means running cargo tests without running code snippets in the doc comments.
Add this because there is a random link time [failure](https://github.com/Phala-Network/phala-blockchain/actions/runs/4422226330/jobs/7753828712) without any useful error message when the CI builds the doc tests.

Reply:
Thx for the explaination